### DISCUSSION TITLE
Will nocodb implement note management features like trilium or obsidian

### DISCUSSION THREAD

**Discussion by AztrexDX** [nocodb/nocodb#6136] [Features]
Will nocodb implement note management features like trilium or obsidian. Like taking notes, linking, running scripts.

**Comment by o1lab** [comment]
@AztrexDX : when you say notes - is it like 'Notes' being a column or like just plain notes instead of a table. Would love to understand more - you can reach out to me [here](https://calendly.com/nocodb-meeting).

**Reply by AztrexDX** [reply]
I was asking about plain notes stored in database. Like trilium stores notes in database. Nocodb stores items in database. If just there would be some way to show these items in hierarchy and would be clear to read
Like some view to show items in hierarchy and on side view the contents of item
![screenshot](https://github.com/nocodb/nocodb/assets/86340924/77b919f2-acf8-4d1b-a2a1-5ce233f010ef)
